scripts/dup_audit.py

```
@@ -0,0 +1,530 @@
+
+def main() -> int:
+    files_paths = list_target_files()
+    file_infos: List[FileInfo] = []
+    for p in files_paths:
+        fi = read_file(p)
+        if fi is None:
+            continue
+        file_infos.append(fi)
+
+    # Restrict to text-like files only for analysis (skip binaries)
+    file_infos = [fi for fi in file_infos if fi.ext in TEXT_EXTS or True]
```

Ruff (0.12.2)

Use `True` instead of `... or True`

Replace with `True`

(SIM222)